src/components/Driver/DriverLogin.js

```diff
@@ -40,24 +40,23 @@ const DriverLogin = () => {
     })
 
     const onClick = (event) => {
-        event.preventDefault();
-        if (fields.password.length < 6) {
-            setFields({ ...fields, alert: 'password length should be at least 6 characters, try again!', password: '' })
-        } else if (!fields.phone_number) {
-            setFields({ ...fields, alert: 'phone is required!' })
-        } else {
-          Login()
-        }
+      event.preventDefault();
+      Login()
```

Comment:
I don't sure if it's okay to make component function asynchronous. Also empty then metod